tests/test_latentdistributiontest.py

```diff
@@ -275,21 +230,30 @@ def test_different_sizes_null(self):
         A1 = er_np(100, 0.8)
         A2 = er_np(1000, 0.7)
 
-        ldt_corrected_1 = LatentDistributionTest(
-            "hsic", "gaussian", n_components=2, n_bootstraps=100, size_correction=True
+        p_corrected_1, _, _ = latent_distribution_test(
+            A1,
+            A2,
+            test="hsic",
+            metric="gaussian",
+            n_components=2,
+            n_bootstraps=100,
+            size_correction=True,
         )
-        ldt_corrected_2 = LatentDistributionTest(
-            "hsic", "gaussian", n_components=2, n_bootstraps=100, size_correction=True
+        p_corrected_2, _, _ = latent_distribution_test(
+            A2,
+            A1,
+            test="hsic",
+            metric="gaussian",
+            n_components=2,
+            n_bootstraps=100,
+            size_correction=True,
         )
 
-        p_corrected_1 = ldt_corrected_1.fit_predict(A1, A2)
-        p_corrected_2 = ldt_corrected_2.fit_predict(A2, A1)
-
         self.assertTrue(p_corrected_1 <= 0.05)
         self.assertTrue(p_corrected_2 <= 0.05)
 
     def test_different_aligners(self):
-        np.random.seed(314)
+        np.random.seed(888)
```

Comment:
f. 

but a serious question is why did you have to reseed? does something break with the old seed?

Reply:
This test kept failing for me so I thought it was originally because of the seed, but it was actually because I was passing input_graph=True where I call latent_distribution_test, so I reverted the seed back

Reply:
oh, okay. just looked suspicious, didn't need to revert back, but leave it now i guess.